docker/pypi/wmagent/bin/manage-common.sh

```diff
@@ -359,8 +383,21 @@ _load_wmasecrets(){
     for varName in $varsToLoad
     do
         value=`grep -E "^[[:blank:]]*$varName=" $secretsFile | sed "s/ *$varName=//"`
-        [[ $varName =~ ^RESOURCE_ ]] && declare -g -A $varName
-        eval $varName=$value
+        if [[ $varName =~ ^RESOURCE_ ]]; then
+	        declare -g -A $varName
+            eval $varName=$value || {
```

Comment:
Same with all this new block of code, some lines are not properly indented and make it a bit harder to read.

Reply:
k

Reply:
Thank you for fixing it. I think the `fi` statement in L399 needs one extra indentation (same level as the else clause)